packages/catalog-search/src/tests/SearchFilters.test.jsx

```diff
@@ -12,6 +12,7 @@ import SearchFilters from '../SearchFilters';
 describe('<SearchFilters />', () => {
   test('renders with a label', () => {
     renderWithSearchContext(
+      // eslint-disable-next-line react/jsx-no-constructed-context-values
       <ResponsiveContext.Provider value={{ width: breakpoints.large.maxWidth }}>
```

Comment:
Other ESLint PRs I looked at recently resolved this error in most tests versus ignoring the ESLint error. Why are we opting to ignore the error here versus fixing it?